readme.html

```diff
@@ -25,3 +25,15 @@
   <li>OAuth (oauth token url, client id and client secret)</li>
   <li>Provided Auth token (auth_token_name, auth_token)</li>
 </ol>
+
+<p><b>put file action</b></p>
+<ul>
+  <li>For the <b>put file</b> action run, we need to start the flask app service on the QA file server to activate the listening port. Steps to run the <b>put file</b> action:</li>
+  <br>
+  <ul>
+  <li>Do the SSH into the QA file server VM with valid credentials</li>
+  <li>Move to flask_app repo</li>
+  <li>Run <b>python cpy.py</b> command to activate the listening port. Make sure service started without any error</li>
+  <li>Go to the Phantom instance and run the action with required parameters. We are using the <b>/web_storage</b> directory instead of <b>/malware_samples</b> due to limited permission to store files on the QA file server</li>
+</ul>
```

Comment:
isn't this the a user-facing README? it seems strange to talk about a `QA file server` in the app's README

Reply:
@rgil-splunk I agree. I guess saying only File Server would be better, or you are suggesting to remove these steps from here?

Reply:
this whole instruction list seems very specific to our use-case (set up VM with credentials to SSH, copy flask_repo, etc.) so I would suggest just removing it. it seems more appropriate for an internal wiki somewhere